docs/source/faq.rst

```diff
@@ -85,14 +85,6 @@ could be other tools that can compile statically typed mypy code to C
 modules or to efficient JVM bytecode, for example, but this is outside
 the scope of the mypy project.
 
-How do I type check my Python 2 code?
-*************************************
-
-You can use a :pep:`comment-based function annotation syntax
```

Comment:
Maybe leave this with an answer like "You can't, use mypy 0.971" or lower.

Reply:
Ehh. mypy will give you that message if you try to use `--python-version 2.7` or synonyms. The people who are trying to type check Python 2 codebases for the first time today do not merit an FAQ entry